### PR TITLE
setting up managed workspace for shebert for demo

### PR DESCRIPTION
**Pull request by scoheb**

### CODE REVIEW

File: components/authentication/tenants/shebert.yaml [hunk 1]
```diff
@@ -10,3 +10,80 @@ roleRef:
...
+    name: scoheb
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: release-service-releasestrategy-editor-role
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: gitopsdeployment-manager
```
Comment (sbose78) [inline components/authentication/tenants/shebert.yaml:43]: ran a search and found we already have something similar - do you want to reconcile these into one?

https://github.com/redhat-appstudio/infra-deployments/blob/9da0f0b67256c53171994a66807cbcce7769119b/components/authentication/tenants/shbose.yaml#L43

Comment (scoheb) [reply components/authentication/tenants/shebert.yaml:43]: sure I can reconcile. Any suggestion as to where this can be extracted to?

Comment (sbose78) [reply components/authentication/tenants/shebert.yaml:43]: Tricky :) How about updating the one that already exists ?

**Review by sbose78 — APPROVED** [review]

**Pull request merged**